pyOCD/tools/flash_tool.py

```diff
@@ -144,12 +154,42 @@ def main():
             if args.hide_progress:
                 progress = None
 
+            # Convert address and count to int.
+            if args.address:
+                args.address = int(args.address, base=0)
+            if args.count:
+                args.count = int(args.count, base=0)
+
+            has_file = args.file is not None
+
             chip_erase = None
             if args.chip_erase:
                 chip_erase = True
             elif args.sector_erase:
                 chip_erase = False
 
+            if not has_file:
+                if chip_erase:
+                    print("Erasing chip...")
+                    flash.init()
+                    flash.eraseAll()
+                    print("Done")
+                elif args.sector_erase and args.address:
+                    flash.init()
+                    page_addr = args.address
+                    for _ in range(args.count):
+                        page_info = flash.getPageInfo(page_addr)
+                        print("Erasing sector 0x%08x" % page_addr)
```

Comment:
You might want to force address to be aligned to a page boundary, and possibly print a warning if it isn't.